src/tests/nativeaot/SmokeTests/UnitTests/Generics.cs

```diff
@@ -20,7 +20,9 @@ internal static int Run()
         TestDelegateVirtualMethod.Run();
         TestDelegateInterfaceMethod.Run();
         TestThreadStaticFieldAccess.Run();
+#if !CODEGEN_WASM // TODO-LLVM: PrecomputedDictionaryLayoutNode.GetSlotForEntry System.InvalidOperationException: [UnitTests]Generics+TestConstrainedMethodCalls.DoFrob<Foo`1<__Canon>,__Canon>(Foo`1<__Canon>&,object): TypeHandle: T_.Foo`1<System.__Canon>
         TestConstrainedMethodCalls.Run();
```

Comment:
Presumably the cause of `.constrained` handling perturbations is https://github.com/dotnet/runtime/pull/66936, but it is not trivial to tell what needs to be changed for things to work again at a glance...

Hopefully we will just be able to re-enable the tests with the IL backend gone.